aten/src/ATen/native/transformers/sdp_utils_cpp.h

```diff
@@ -22,13 +22,14 @@
 
 namespace sdp {
 
-constexpr int32_t num_backends = 4;
+constexpr int32_t num_backends = 5;
 enum class SDPBackend {
   error = -1,
   math = 0,
-  flash_attention = 1,
-  efficient_attention = 2,
-  cudnn_attention = 3
+  overrideable = 1,
```

Comment:
can you add this backend to the end of the enum list, this is potentially bc breaking, especially since our bindings for enums often desugar to ints

Reply:
ok

Reply:
@drisspg Looking forward to your reply.